src/app/connector-editor/components/connector-editor/connector-editor.component.ts

```diff
@@ -99,16 +101,4 @@ export class ConnectorEditorComponent {
             .select(selectSelectedTheme)
             .pipe(map(theme => (theme.className === 'dark-theme' ? 'vs-dark' : 'vs-light')));
     }
-
-    private getEditorContent(): Observable<string> {
```

Comment:
This one is breaking a current feature of the connector editor. Since Travis was not complaining, I'd like to ask you to write an e2e test for this feature. We can discuss the details in call